### PR TITLE
scrub keywords during filter rendering

### PR DESCRIPTION
**Pull request by danlipert**
##### Description

This PR solves an issue where untrusted content from the URL bar can be rendered unsafely into the DOM

##### Checklist

- [X] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [X] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [X] Linter status: 100% pass
- [ ] Added relevant tests
- [X] Tests pass and code coverage has not decreased
- [ ] Changes don't break existing behavior
- [X] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [X] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

##### Affected core subsystem(s)

bounties explorer

##### Testing and Sign-off

tested locally against disclosed issue

###### Contributor

- [X] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [X] Tested all changes **locally**
- [X] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

File: app/assets/v2/js/pages/dashboard.js [hunk 1]
```diff
@@ -74,6 +74,10 @@ function debounce(func, wait, immediate) {
   };
 }
 
+var scrub = function(v) {
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/dashboard.js]: thought on updating it to `const scrub = value => value.replace(/[\W]+/g, '')` ?

**Comment by danlipert** [comment]
> could we update the function definition to ES6 format? LGTM otherwise

will do! I'll rewrite it later tonight 👍

**Comment by octavioamu** [comment]
LGTM but we are already loading https://github.com/cure53/DOMPurify maybe worst using it instead.

**Comment by danlipert** [comment]
@thelostone-mc updated! 👍

**Pull request merged**